package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cowprotocol/cow-sdk",
-  "version": "4.1.0",
+  "version": "5.0.0",
```

Comment:
will this affect to anything in learn.cow.fi?

Reply:
There is no use of polling yet in any tutorials there.